### DISCUSSION TITLE
Restricting Minimum Width and Height During Resize with Halo in JointJS

### DISCUSSION THREAD

**Discussion by VADDIJASWANT** [clientIO/joint#2416] [Q&A]
### Introduction

Hello JointJS Community,

In my project, I’m trying to restrict the resizing of elements to have a minimum width and height. I noticed that when elements are in selection, they automatically have the minimum height and width of the grid size during resizing. However, this doesn’t seem to work with the resize option in halo icons.

I’m looking for a way to add this restriction to the halo resize function.

I’ve attached a small video for reference to better illustrate the issue. Any guidance on how to achieve this would be greatly appreciated.

Thank you in advance for your help!

https://github.com/clientIO/joint/assets/116190897/0ee9af90-4213-40b5-9054-07ca5d9c7570

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment]
It's currently not possible to set the `minWidth` and `minHeight` for the `ui.Halo` resize.

You can remove the `resize` handle and use the [Control](https://resources.jointjs.com/docs/jointjs/v3.7/joint.html#elementTools.Control) element tool instead. See [example](https://codepen.io/jointjs/pen/ZERyVxe).
Also the `ui.FreeTransform` allows you to define `minWidth` and `minHeight` (and more) when resizing.
